auto_nag/scripts/component.py

```diff
@@ -142,7 +136,7 @@ def get_bugs(self, date="today", bug_ids=[]):
 
             result = {
                 "id": bug_id,
-                "summary": self.get_summary(bug),
+                "summary": bug["summary"],
```

Comment:
I think we need to keep using get_summary (as the summary will be hidden for sec bugs)

Reply:
The "summary" we get is already the result of `get_summary` according to https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/bzcleaner.py#L246